.github/workflows/main.yaml

```diff
@@ -221,3 +221,20 @@ jobs:
           fetch-depth: 0
 
       - run: make image
+
+  buildx:
+    runs-on: ubuntu-18.04
+    timeout-minutes: 30
+    name: HHHHHHHHHHHHHHHHHHHHHHHHHHHHHH Buildx to build images
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - uses: docker/setup-buildx-action@v1
+        id: buildx
+        with:
+          install: true
+      - name: Build
+        run: |
+          docker build -f /home/runner/work/${{ github.repository }}/build/cloud/Dockerfile # will run buildx
+        with:
+          platforms: linux/amd64,linux/arm64,linux/arm/v7
```

Comment:
Should do the make command from makefile.

Reply:
got it, i'll integrate with buildx in https://github.com/kubeedge/kubeedge/pull/3530